docs/architecture/interconnect.rst

```diff
@@ -0,0 +1,26 @@
+Interconnect PIPs
```

Comment:
Link first use of "PIP" to the glossary.

Reply:
Added references to glossary.